src/Robo/Plugin/Commands/CICommands.php

```diff
@@ -118,34 +116,31 @@ public function jobCheckCodingStandards(): Result
      */
     public function jobFixCodingStandards(): Result
     {
-        $standards = implode(',', $this->getCodingStandards());
-        $extensions = implode(',', $this->phpcsCheckExtensions);
-        $ignorePaths = implode(',', $this->phpcsIgnorePaths);
-        /** @var \Robo\Task\CommandStack $stack */
-        $stack = $this->taskExecStack()->stopOnFail();
-        $stack->exec("vendor/bin/phpcbf --standard=$standards --extensions=$extensions \
-                --ignore=$ignorePaths $this->customCodePaths");
-        if ($this->lintTwigFiles) {
-            $stack->exec("vendor/bin/twig-cs-fixer lint $this->customCodePaths --fix");
-        }
-
-        return $stack->run();
+        return $this->jobRunCodingStandards(true);
     }
 
     /**
-     * Get custom code paths for static analysis tools to use.
+     * Determine whether to check or fix coding standard issues and run as requested.
      *
-     * @return array
-     *   An array containing application custom code paths.
+     * @param bool $applyFixes
+     *   FALSE if phpcs should be run, else phpcbf.
      *
-     * @throws \Robo\Exception\TaskException
+     * @return \Robo\Result
+     *   The result of the set of tasks.
      */
-    protected function getCustomCodePaths(): array
+    protected function jobRunCodingStandards(bool $applyFixes = false): Result
```

Comment:
I'm on the fence about this consolidation. I see pros and cons, but not enough either way to take a stand right now.

So 🎉 I guess? haha.